packages/hint-create-element-svg/README.md

```diff
@@ -0,0 +1,80 @@
+# create-element-svg (`@hint/hint-create-element-svg`)
```

Comment:
This title should align better with our other hints (probably another place `create-hint` could use an update).
```suggestion
# No `createElement` with SVG (`create-element-svg`)
```